resources/views/tasks/preview.blade.php

```diff
@@ -339,7 +339,7 @@ class="card border-0"
           },
           submit(task) {
             if (this.isSelfService) {
-              ProcessMaker.alert(this.$t('Claim the Task to continue.'), 'warning');
+              window.parent.ProcessMaker.alert(this.$t('Claim the Task to continue.'), 'warning');
```

Comment:
Please create a method in order to define in one place the this line and only call the method

Reply:
@pmPaulis , Actually ProcessMaker.alert is already a method. With this solution the message will be showed in the Processmaker and not in the frame
cc @luNunezProcessmaker